actions/tor

```diff
@@ -98,6 +100,8 @@ def subcommand_setup(_):
 
     aug.save()
 
+
+    subprocess.run(['tor-instance-create', 'plinth'])
```

Comment:
This needs to be run before all the augeas settings above. Otherwise the config file that we are trying to modify, doesn't exist yet.